src/pages/home/ReportScreen.js

```diff
@@ -420,7 +432,7 @@ function ReportScreen({
                             >
                                 {isReportReadyForDisplay && !isLoadingInitialReportActions && !isLoading && (
                                     <ReportActionsView
-                                        reportActions={reportActions}
+                                        reportActions={reportActionToDisplay}
```

Comment:
We also have `reportActions` pass to `ReportFooter`; why do we only change this component?